versioned_docs/version-0.9.1/Installation/additional-configuration.md

```diff
@@ -16,77 +16,90 @@ import TOCInline from '@theme/TOCInline';
+- Your bot's token
+- Your Discord Guild ID (optional, but useful if you want the user to be added to your server)
+- Your Discord Invite (optional, but useful if you want the user to be added to your server)
+- Your Discord Role ID (optional, but useful if you want the verified user to recieve a role on your server after verifying)
+3. Then press the "Submit" button to save.
 
-That's it! The Dashboard can now use your bot to make API calls to Discord to place new users into your Discord server. Your bot will not come online as we only use it to make API calls when someone verifies, you can still use your bot as you normally would :)
-
-**Custom Role Reward**<br/>
-If you would like to give your users a custom role automatically when they verify their Discord account, you have to copy the ID of an existing role from your Discord server and place it on the webpage as well.
+That's it! The Dashboard can now use your bot to make API calls to Discord to place new users into your Discord server. Your bot will not come online as we only use it to make API calls when someone verifies, you can still use your bot as you normally would as it won't affect the bot itself.
 
 ### reCAPTCHA v2
 
 Go to `https://<your.ctrlpanel.gg>/admin/settings#misc`
 
-Controlpanel uses visible reCAPTCHA to secure the login page from brute-force attacks. If the login attempt is considered suspicious, users may be required to perform a reCAPTCHA challenge.
+Ctrlpanel uses visible reCAPTCHA to secure the login page from brute-force attacks. If the login attempt is considered suspicious, users may be required to perform a reCAPTCHA challenge.
 
 You can generate your own keys in the [admin console](https://www.google.com/recaptcha/admin/create)
 
-To add your keys, you need to add them on the webpage.
+Go to the Admin Misc settings and change the `reCAPTCHA Site-Key` and `reCAPTCHA Secret-Key`. Once you've updated it, press `Submit`. If it works correctly then below the reCAPTCHA settings under `Your reCAPTCHA`, you should be able to see the reCAPTCHA without the `This reCAPTCHA is for testing purposes only. Please report to the site adminif you are seeing this`.
+
+**FAQ for reCAPTCHA**<br/>
+
+**Q:** It says `ERROR for site owner: Invalid site key` <br/>
+**A:** This means that you put in the incorrect site key. Double check you're using v2 and `I'm not a robot` checkbox on the reCAPTCHA create page which is found [here](https://www.google.com/recaptcha/admin/create)
 
 ### Manual Mail Configuration
 
 To send out notifications and mails, you will need to configure your mail variables correctly.
-You can use a free mail server; for example, from [Mailjet](https://www.mailjet.com/). You can also install your own mail server using, for example, [mailcow](https://mailcow.email/)
+You can use a free mail server; for example, from [Zoho](https://www.zoho.com/mail). You can also install your own mail server using, for example, [Mailcow](https://mailcow.email/)
```

Comment:
https://www.zoho.com/mail isn´t free (or at least i don´t see it - Cheapest is 0.9€/M) and it seems to be a email host rather than just a email service.

Reply:
ZM was free when I last tried it, but it did not support SMTP

Reply:
@haydenwalker980 it does for me when i link my domain for free, still does to this day. however ill revert it to mailjet for now